config/environments/production.rb

```diff
@@ -64,13 +64,14 @@
   # set a host name for emails
   config.action_mailer.default_url_options = { :host => 'vpb.concord.org' }
 
-  # this is the way to do exception notification now in Rails 3
-  config.middleware.use ExceptionNotifier,
-    :email_prefix => "[VPB Exception] ",
-    :sender_address => %{"Exception Notifier" <vpb-errors@concord.org>},
-    :exception_recipients => %w{vpb-errors@concord.org}
-
```

Comment:
NOTE: this was removed in Rails 4, did not catch until now as the production environment was not used until now.